src/vue.config.js

```diff
@@ -0,0 +1,9 @@
+module.exports = {
+    css: {
+        loaderOptions: {
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 8.](https://app.codacy.com/gh/Supershift/Sushi.Mediakiwi.UI/pullRequest?prid=8211123)
```suggestion
    loaderOptions: {
```